config/class-site-details-index.php

```diff
@@ -244,6 +247,40 @@ public function get_parsely_info() {
 		return $parsely_info;
 	}
 
+	public function get_extra_blog_meta() {
+		$blog_meta = [];
+
+		// Block Editor
+		$is_using_block_editor = $this->is_using_block_editor();
+		$blog_meta[] = [
+			'name' => 'using_block_editor',
+			'value' => $is_using_block_editor ? 1 : 0,
+		];
+		
+		return $blog_meta;
+	}
+
+	// Somewhat naive detection of block editor use
+	private function is_using_block_editor() {
+		// TODO: probably move this elsewhere
+		$default_post = get_default_post_to_edit();
+
+		$most_recent_post_array = get_posts( [
+			'post_type' => $default_post->post_type,
+			'post_status' => 'publish',
+			'posts_per_page' => 1,
+			'orderby' => 'post_date',
+			'order' => 'DESC',
+		] );
+
+		$is_using_block_editor = false;
+		if ( ! empty( $most_recent_post_lookup ) ) {
```

Comment:
```suggestion
		if ( ! empty( $most_recent_post_array ) ) {
```

`$most_recent_post_lookup` does not exist.